app/preload/database/identityDao.ts

```diff
@@ -18,18 +18,28 @@ import {
 } from '~/utils/types';
 import { IdentityMethods } from '~/preload/preloadTypes';
 
+interface MaxIdentityNumber {
+    maxIdentityNumber: number;
+}
+
 /**
  * Get the identity number to be used to create the next identity with
  * the wallet with the given id.
  * @param walletId the database id key for the wallet used
  * @returns the id for the next identity to be created by the given wallet
  */
 export async function getNextIdentityNumber(walletId: number): Promise<number> {
-    const model = (await knex())
-        .table(identitiesTable)
-        .where('walletId', walletId);
-    const totalCount = await model.clone().count();
-    return parseInt(totalCount[0]['count(*)'].toString(), 10);
+    const maxIdentityNumber = await (await knex())
+        .table<Identity>(identitiesTable)
+        .where('walletId', walletId)
+        .max<MaxIdentityNumber>('identityNumber as maxIdentityNumber')
```

Comment:
```suggestion
        .max<{
            maxIdentityNumber: number;
        }>('identityNumber as maxIdentityNumber')
```

You can inline the type, instead of creating an interface for an inline variable.